tests/test_agents/base.py

```diff
@@ -216,20 +216,13 @@ def __set_configs(self, i: int, nb_agents: int) -> None:
         self.__set_extra_configs()
 
     @staticmethod
-    def __get_agent_name(i: int, nb_agents: int) -> str:
+    def _get_agent_name(i: int) -> str:
         """Get the ith agent's name."""
-        if i < 0:
-            i = nb_agents + i
-        if i < 0:
-            raise ValueError(
-                f"Incorrect negative indexing. {i} was given, but {nb_agents} agents are available!"
-            )
-        agent_name = f"agent_{i:05d}_{nb_agents}agents_run"
-        return agent_name
+        return f"agent_{i:05d}"
```

Comment:
why are we no longer logging total number of agents too?

Reply:
because it is defined at the test level, it will be reported in the name of the test already and adding it to the agent name itself we need to pass around another variable